www/src/scss/navigator/misc/_misc.scss

```diff
@@ -116,55 +116,62 @@ $misc-footer-active: 70px;
         }
       }
     }
-    .announcement-box {
-      @include flex($fd-value: column);
-      header {
-        border-top: 1px #757575 solid;
-        color: #757575;
-        font-size: 0.8em;
-        font-weight: bold;
-        letter-spacing: 0.2em;
-        margin-bottom: 0.5em;
-        margin-left: 1rem;
-        margin-top: 1em;
-        padding-top: 0.75em;
-        word-spacing: 0.2em;
-        cursor: default;
-        text-transform: uppercase;
-      }
-      .announcement-switch {
-        @include flex($jc-value: space-evenly, $width: initial);
-        padding: 5px;
-        span {
-          padding-right: 3px;
+    .announcement {
+      &-box {
+        @include flex($fd-value: column);
+        gap: 22px;
+        padding: 22px 12px;
+
+        header {
+          color: #757575;
+          font-size: 0.8em;
+          font-weight: bold;
+          letter-spacing: 0.15em;
+          text-align: center;
+          text-transform: uppercase;
+          word-spacing: 0.15em;
+          width: 80%;
+
+          i {
+            padding-right: 6px;
+          }
         }
       }
-      .announcement-text {
+      &-switch {
+        @include flex($jc-value: space-between, $ai-value: center, $width: 100%);
+      }
+      &-text {
         border-radius: 10px;
         display: inline-block;
-        margin: 0 20px 10px;
         min-height: 100px;
         min-width: 300px;
         outline: none;
         padding: 10px;
       }
-      .announcement-slide-btn {
+      &-slide-btn {
         align-self: center;
+        background: #29679b;
         border: 0;
         border-radius: 0;
         font-size: 0.9rem;
         color: #fff;
```

Comment:
Not a edit in this PR, but came to my attention, we do have a variable for white color which can be used here.

Reply:
Done veerji, Also there was one more hex in this file I have changed that to variable as well. Thanks for pointing out